Jenkinsfile

```diff
@@ -178,6 +178,11 @@ node('ubuntu-zion') {
         }
         OsTools.runSafe(this, "git tag -d ${version}")
       }
+
+      //Trigger the Red Hat job and It does not care about this result
+      stage('Trigger Red Hat Certified Job Build') {
+        build(job: '/integrations/Red Hat Certified Docker Image/iq-redhat-certified', wait: false)
```

Comment:
Might also want propagate:false although I'm not certain how that interacts with wait. Pretty sure we don't want this tied to the downstream build result in any case

Reply:
Right, if the intent is 'fire and forget' then you don't want the downstream status to propagate back to this job.

https://www.jenkins.io/doc/pipeline/steps/pipeline-build-step/#build-build-a-job